Documentation/ceph-pool-crd.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Ceph Pool
-weight: 34
+weight: 33
```

Comment:
shall we similarly change the ceph weights under the `Storage` node so they don't skip numbers? And do we need to add a `Ceph` prefix to the "Block", "Object", and "File" storage table of content items, similar to the CRDs? Though I can't wait to see the docs factored out to have more independent storage provider docs.